include/seqan3/search/algorithm/detail/search_trivial.hpp

```diff
@@ -49,13 +56,14 @@ namespace seqan3::detail
  */
 template <bool abort_on_hit, typename query_t, typename cursor_t, typename delegate_t>
 inline bool search_trivial(cursor_t cur, query_t & query, typename cursor_t::size_type const query_pos,
-                           search_param const error_left, delegate_t && delegate) noexcept(noexcept(delegate))
+                           search_param const error_left, last_error const memory,
+                           delegate_t && delegate) noexcept(noexcept(delegate))
 {
     // Exact case (end of query sequence or no errors left)
     if (query_pos == std::ranges::size(query) || error_left.total == 0)
     {
         // If not at end of query sequence, try searching the remaining suffix without any errors.
-        if (query_pos == std::ranges::size(query) || cur.extend_right(view::drop(query, query_pos)))
+        if (query_pos == std::ranges::size(query) || cur.extend_right(std::ranges::view::drop(query, query_pos)))
```

Comment:
Why this change?

Reply:
Travis failed: https://travis-ci.org/seqan/seqan3/jobs/563954977
with: 
In file included from /home/travis/build/seqan/seqan3-build/seqan3_header_test_files/seqan3/search/algorithm/detail/search_trivial-2.cpp:2:0:

/home/travis/build/seqan/seqan3/include/seqan3/search/algorithm/detail/search_trivial.hpp: In function ‘bool seqan3::detail::search_trivial(cursor_t, query_t&, typename cursor_t::size_type, seqan3::detail::search_param, seqan3::detail::last_error, delegate_t&&)’:

/home/travis/build/seqan/seqan3/include/seqan3/search/algorithm/detail/search_trivial.hpp:66:71: error: ‘view’ has not been declared

         if (query_pos == std::ranges::size(query) || cur.extend_right(view::drop(query, query_pos)))

What confusing it only started to fail recently like 1.5 weeks ago before this was fine for travis.

Reply:
Revert this and revert the exclusion of the include:
```cpp
#include <seqan3/range/view/drop.hpp>
```